webrtc.html

```diff
@@ -2075,6 +1826,314 @@
+                requirements for sending (see <a href="#isolated-pc">
+                isolated streams and RTCPeerConnection</a>).</p>
+
+                <p>All other tracks that are not accessible to the application
+                MUST NOT be sent to the peer, with silence (audio), black
+                frames (video) or equivalently absent content being sent in
+                place of track content.</p>
+
+                <p>Note that this property can change over time.</p>
+              </li>
+
+              <li>
+                <p>If <var>connection</var>'s <a href=
+                "#dom-peerconnection-signaling-state"><code>RTCPeerConnection</code>
+                signalingState</a> is <code>stable</code>, then fire a <a href=
+                "#event-negotiation">negotiationneeded</a> event at
+                <var>connection</var>.</p>
+              </li>
+            </ol>
+          </dd>
+
+          <dt>void removeTrack (RTCRtpSender sender)</dt>
+
+          <dd>
+            <p>Removes <var>sender</var>, and its associated <code>MediaStreamTrack</code>, from the
+            <code><a>RTCPeerConnection</a></code>.</p>
+
+            <!-- onended -->
+            <p>When the other peer stops sending a track in this manner, a
+            <code title="event-MediaStreamTrack-ended"><a href=
+            "#event-mediastream-ended">ended</a></code> event is
+            fired at the <code><a>MediaStreamTrack</a></code> object.</p>
+
+            <p>When the <dfn id="dom-peerconnection-removetrack"><code title=
+            "">removeTrack()</code></dfn> method is invoked, the user agent
+            MUST run the following steps:</p>
+
+            <ol>
+              <li>
+                <p>Let <var>connection</var> be the
+                <code><a>RTCPeerConnection</a></code> object on which the
+                <code><a>RTCRtpSender</a></code>, <var>sender</var>, is to be
+                removed.</p>
+              </li>
+
+              <li>
+                <p>If <var>connection</var>'s <a href=
+                "#dom-peerconnection-signaling-state"><code>RTCPeerConnection</code>
+                signalingState</a> is <code>closed</code>, throw an
+                <code>InvalidStateError</code> exception.</p>
+              </li>
+
+              <li>
+                <p>If <var>sender</var> is not in <var>connection</var>'s
+                <a href="#senders-set">set of senders</a>, then abort
+                these steps.</p>
+              </li>
+
+              <li>
+                <p>Remove <var>sender</var> from <var>connection</var>'s
+                <a href="#senders-set">set of senders</a>.</p>
+              </li>
+
+              <li>
+                <p>If <var>connection</var>'s <a href=
+                "#dom-peerconnection-signaling-state"><code>RTCPeerConnection</code>
+                signalingState</a> is <code>stable</code>, then fire a <a href=
```

Comment:
This does not seem right in the case things are not in stable. The event is still needed but perhaps it happens later.